src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceCancelTaskTest.java

```diff
@@ -71,79 +64,6 @@ void setUp() {
         );
     }
 
-    @ParameterizedTest(name = "{0}")
-    @MethodSource("getGrantTypes")
-    void should_retrieve_a_task_to_cancel(GrantType grantType) {
-        final String taskId = "8d6cc5cf-c973-11eb-bdba-0242ac111017";
-        final String caseId = "1623278362431017";
-
-        List<RoleAssignment> roleAssignments = new ArrayList<>();
-
-        Map<String, String> tcAttributes = new HashMap<>();
-
-        if (grantType == GrantType.SPECIFIC) {
-            tcAttributes.put(RoleAttributeDefinition.JURISDICTION.value(), IA_JURISDICTION);
-            tcAttributes.put(RoleAttributeDefinition.CASE_TYPE.value(), IA_CASE_TYPE);
-            tcAttributes.put(RoleAttributeDefinition.CASE_ID.value(), caseId);
-        } else if (grantType == GrantType.STANDARD) {
-            tcAttributes.put(RoleAttributeDefinition.JURISDICTION.value(), IA_JURISDICTION);
-            tcAttributes.put(RoleAttributeDefinition.CASE_TYPE.value(), IA_CASE_TYPE);
-            tcAttributes.put(RoleAttributeDefinition.CASE_ID.value(), caseId);
-            tcAttributes.put(LOCATION.name(), "1");
-        }
-
-        RoleAssignment roleAssignment = RoleAssignment
-            .builder()
-            .roleName("tribunal-caseworker")
-            .roleType(RoleType.ORGANISATION)
-            .classification(Classification.PUBLIC)
-            .beginTime(LocalDateTime.now().minusYears(1))
-            .endTime(LocalDateTime.now().plusYears(1))
-            .grantType(grantType)
-            .attributes(tcAttributes)
-            .build();
-        roleAssignments.add(roleAssignment);
-
-        AccessControlResponse accessControlResponse = new AccessControlResponse(null, roleAssignments);
-        permissionsRequired.add(PermissionTypes.CANCEL);
-
-
-        final Optional<TaskResource> task = cftQueryService.getTask(taskId, roleAssignments, permissionsRequired);
-        Assertions.assertThat(task.isPresent()).isTrue();
-        Assertions.assertThat(task.get().getTaskId()).isEqualTo(taskId);
-        Assertions.assertThat(task.get().getCaseId()).isEqualTo(caseId);
-    }
-
-    @Test
-    void should_retrieve_a_task_to_cancel_challenged() {
```

Comment:
1. Test data setup has commonality/duplicates for getTask(...), however the permission Types differ across the various actions (This one is Cancel). Is this really a duplicate?
2. Looks like we have removed this because it's IA centric. Does that mean we have the WA equivalent?

Reply:
wa/CftQueryServiceCancelTaskTest
should_retrieve_a_task_when_grant_type_challenged